tests/admin_views/models.py

```diff
@@ -582,6 +582,17 @@ class Meta:
         proxy = True
 
 
+class Student(models.Model):
```

Comment:
```
class Topping(models.Model):
    name = models.CharField(max_length=20)

    def __str__(self):
        return self.name


class Pizza(models.Model):
    name = models.CharField(max_length=20)
    toppings = models.ManyToManyField('Topping', related_name='pizzas')
```
It looks like these existing models are equivalent. If needed, you can create a proxy model of one or more of them as needed, then register those with a separate ModelAdmin. (15b465c584f49a1d43b6c18796f83521ee4ffc22 is a past example of doing that.)

Reply:
I removed all the models I added and only added Song. That should be better I presume.

Reply:
I tried to remove as many models as I can. Should be better now